src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -739,17 +746,20 @@ void MapSelect_Init(GameState* thisx) {
     this->lockDown = 0;
     this->unk_234 = 0;
 
-    size = (uintptr_t)_z_select_staticSegmentRomEnd - (uintptr_t)_z_select_staticSegmentRomStart;
-
     if ((dREG(80) >= 0) && (dREG(80) < this->count)) {
         this->currentScene = dREG(80);
         this->topDisplayedScene = dREG(81);
         this->pageDownIndex = dREG(82);
     }
+
     R_UPDATE_RATE = 1;
 
-    this->staticSegment = GAME_STATE_ALLOC(&this->state, size, "../z_select.c", 1114);
-    DMA_REQUEST_SYNC(this->staticSegment, (uintptr_t)_z_select_staticSegmentRomStart, size, "../z_select.c", 1115);
+    {
+        u32 size = (uintptr_t)_z_select_staticSegmentRomEnd - (uintptr_t)_z_select_staticSegmentRomStart;
+        this->staticSegment = GAME_STATE_ALLOC(&this->state, size, "../z_select.c", 1114);
```

Comment:
whenever a new stack variable is declared, we prefer to include a new line before the rest of the code. so you can add a break after the size line